<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab_input_info">
       <attribute name="title">
        <string>信息录入</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </widget>
      <widget class="QWidget" name="tab_check">
       <attribute name="title">
        <string>试卷批改</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5"/>
      </widget>
      <widget class="QWidget" name="tab_analysis">
       <attribute name="title">
        <string>成绩分析</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7"/>
      </widget>
      <widget class="QWidget" name="tab_export">
       <attribute name="title">
        <string>导出成绩</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
